exonum-java-binding/package_app.sh

```diff
@@ -92,6 +92,9 @@ mkdir -p "${PACKAGING_BASE_DIR}"
 mkdir -p "${PACKAGING_NATIVE_LIB_DIR}"
 mkdir -p "${PACKAGING_ETC_DIR}"
 
+# Enable static linkage for RocksDB
+export ROCKSDB_STATIC=1
```

Comment:
Is there any interference with `ROCKSDB_LIB_DIR` that is likely to be set? Shall we unset it? Or it won't hurt?

Reply:
Good catch. Actually it will work _only_ if `ROCKSDB_LIB_DIR` is set